novawallet/Modules/BackupAttention/BackupAttentionViewLayout.swift

```diff
@@ -1,6 +1,34 @@
 import UIKit
+import SoraUI
 
 final class BackupAttentionViewLayout: UIView {
+    var checkBoxScrollableView = CheckBoxIconDetailsScrollableView()
+
+    var agreeButton: UIControl?
+
+    var blurredBottomView: OverlayBlurBackgroundView = .create { view in
```

Comment:
Please, check StartStakingInfoViewLayout for the reference. It uses BlurBackgroundView to setup similar footer